content/2_Bronze/Intro_Sorting.mdx

```diff
@@ -121,22 +146,19 @@ Although you usually do **not** need to know how sorting is implemented, you sho
 
 </LanguageSection>
 
-### (Dynamic) Arrays
+### Static Arrays
```

Comment:
It would be good if you added code snippets + output (like what was done for "(Dynamic) Arrays of Pairs & Tuples")

Reply:
Of course, the linked resources cover this, but I think it's helpful to have a code snippet that actually runs instead of just `sort(arr, arr + N)`